ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-hardware.j2

```diff
@@ -0,0 +1,26 @@
+{% if ip_hardware is arista.avd.defined %}
+
+## IP Hardware
+{%   if ip_hardware.fib is arista.avd.defined %}
```

Comment:
Indentation should be 4 spaces. There is a CI check failing due to indentation. Please make sure indentation at all if statements.

Run ```pre-commit run -a``` before pushing any changes to make sure you catch these errors locally.

Reply:
these will only be caught by our `j2lint` library.